docs/explanations/how_to_interpret_output_of_fix_plan_api.md

```diff
@@ -0,0 +1,222 @@
+             will be detailed in a following section.
+* **expected_status**: The expected status of the CVE if the plan was to be executed. There are
+  three possible scenarios: **fixed**, **still-affected** and **not-affected**.
+  The system is considered **still-affected** if there is something that
+  prevents any required packages to be upgraded. Additionally, the system
+  is considered **not-affected**  if the CVE doesn't affect the system at all.
+* **plan**: The necessary plan that would need to be executed to fix the CVE. We will detail each
+  possible plan step on a following section.
+* **warnings**: Warnings that can happen when generating the plan. For example, if the CVE doesn't
+  have a fix available for a given package, this will be reflected here. We will detail the possible
+  warning in a following section.
+
+
+## What operations can appear in the plan array ?
+
+There are four distinct operations that can appear on the plan array. They are:
+
+* **APT Upgrade**: This operation details which package would need to be upgraded to fix the CVE.
+  The JSON representation for this step is the following one:
+
+  ```json
+  {
+    "data": {
+      "binary_packages": ["binary_package"],
+      "source_packages": ["source_package"]
+    },
+    "operation": "apt-upgrade",
+    "order": 1
+  }
+  ``` 
+
+  This step provide all the binary and source packages on that data object.
+
+* **Attach**: This operation details that the user will need to attach to a Pro subscription.
+  The JSON representation for this step is the following one:
+
+  ```json
+  {
+    "data": {
+      "reason": "required-pro-service"
+    },
+    "operation": "attach",
+    "order": 1
+  }
+  ```
+
+  The reason field details why an attach is needed. This can be either because the user has
+  an expired subscription or the fix requires a specific Pro service to upgrade an affected
+  package.
+
+* **Enable**: This operation details that the user needs to enable a specific Pro service.
+  The JSON representation for this step is the following one:
+
+  ```json
+  {
+    "data": {
+      "service": "esm-infra"
+    },
+    "operation": "enable",
+    "order": 1
+  }
+  ```
+
+  The data object contains which service will need to be enabled.
+
+* **NoOp**: This indicates that no operation need to be performed to address the CVE.
+  The JSON representation for this step is the following one:
+
+  ```json
+  {
+    "data": {
+      "status": "system-not-affected"
+    },
+    "operation": "no-op",
+    "order": 1
+  }
+  ```
+  
+  The data object will state why no operation is needed. This can either be because the CVE doesn't
+  affect the system or because the CVE is already fixed in the machine.
+
+## What type of warnings can be generated ?
+
+There are two distinct warning that can happen when executing the plan API:
+
+* **Package cannot be installed**: This happens when the endpoint identifies that a given package
```

Comment:
As per my comment above, I think it'd be helpful to turn these bolded bits into sub-subheadings instead